### ISSUE TITLE
Error installing- ENOENT: no such file or directory, open '/home/pi/package.json'

### ISSUE THREAD

**Issue by Kepete** [ageorgios/homebridge-sonoff-tasmota-http#8]
When installing get these warnings: 

<img width="595" alt="screen shot 2018-01-30 at 23 24 36" src="https://user-images.githubusercontent.com/32489848/35592214-fd689406-0614-11e8-9dce-fe20a5327bb2.png">

At /home/pi/package.json there is no file named package.json

When I try to run Homebridge it says `Error: The requested accessory 'SonoffTasamotaHTTP' was not registered by any plugin.` So the plugin didn't install correctly.

**Comment by Kepete** [comment]
Before I posted this issue I did the whole day research and didn't figure it out. Just after writing this issue I resolved it myself. A noobs mistake, but might be useful for some other beginners. 
I first tried to install the plugin in the /home/pi/ folder. This is wrong, it has to go to the node app root directory. You can find it by `npm -g root` command. 

As I installed half of it(54 packages were added) into the /home/pi/ , is there a way to uninstall it from there and only leave it where it should be?

**Comment by Kepete** [comment]
As I got it installed now I'm running into another issue, might be a noobs problem but still. 
I can see the accessory in my home app on iPhone, but it is no responding. I can't toggle it on and off.

**Comment by ageorgios** [comment]
What version are you on?

The Tasmota compatible version of the plugin is 5.11.0 and later

If you need compatibility with previous Tasmota versions, fork this commit: https://github.com/ageorgios/homebridge-sonoff-tasmota-http/tree/6f73a32fd8ae01f16813f8f0bd3844d3da469e4d

**Comment by Kepete** [comment]
@ageorgios I'm running Sonoff-Tasmota 5.11.1

**Comment by ageorgios** [comment]
run this on rpi
sudo journalctl -u homebridge -f

try to turn on-off
and check the log why is it not working

**Comment by Kepete** [comment]
It doesn't give any information
`-- No entries --`

**Comment by ageorgios** [comment]
I assumed you have rpi with jessie installed.
Try to find the logs of the service of homebridge

**Comment by Kepete** [comment]
Found the logs. 
I started the logging, run Homebridge and tried to turn the Sonoff on and off from my phone.  Nothing about the Sonoff is showing up on the logs. 

<img width="1276" alt="screen shot 2018-01-31 at 11 29 39" src="https://user-images.githubusercontent.com/32489848/35615191-16db1372-067a-11e8-86fc-34c360999b1f.png">

**Comment by ageorgios** [comment]
The log usually writes more information,
maybe it is because the session is closed for user root

**Comment by Kepete** [comment]
@ageorgios  Had time today to play around again. 
The logs display the session closed message after I close the logging. 
I couldn't figure out how to get more log info than that. 

I would like to go over the setup that I did everything correctly. 
- Installed the plugin in the /home/pi/ folder
- Edited the config.json file. Replaced there the `"The hostname of the Sonoff device"` with the hostname what I found in the Sonoff web interface. 
- Run Homebridge and see that Homebridge has initialised the plugin 
- Home app on iPhone shows the Sonoff accessory, but says still `No response`
